src/controller/python/OpCredsBinding.cpp

```diff
@@ -319,6 +322,68 @@ void pychip_OnCommissioningStatusUpdate(chip::PeerId peerId, chip::Controller::C
     return sTestCommissioner.OnCommissioningStatusUpdate(peerId, stageCompleted, err);
 }
 
+/**
+ * By using pychip_OpCreds_AllocateControllerForCustomCommissioningFlow, we will not using the commissioning flow from the
+ * Controller class.
+ *
+ */
+PyChipError pychip_OpCreds_AllocateControllerForCustomCommissioningFlow(chip::Controller::DeviceCommissioner ** outDevCtrl,
+                                                                        uint8_t * serializedEphemeralKey,
```

Comment:
We need to pass a P256Keypair here, not raw bytes

Reply:
already resolved